rct229/ruleset_functions/baseline_systems/test_is_baseline_system_8.py

```diff
@@ -0,0 +1,223 @@
+from rct229.schema.validate import schema_validate_rmr
+
```

Comment:
I was not being consistent on this but it will be helpful to start documenting the unit tests at the top of the file. One example is:
[Sys type 3](https://github.com/pnnl/ruleset-checking-tool/blob/develop/rct229/ruleset_functions/baseline_systems/test_is_baseline_system_3.py)

Reply:
Great point! Fixed.